tests/__snapshots__/LivewireTest__it_will_add_comments_for_Livewire_components__1.html

```diff
@@ -5,19 +5,16 @@
 <div>
     This is the Livewire component
 </div>
-
-<!-- Livewire Component --><!-- End Livewire component: test -->
```

Comment:
Seems like it existed in version 2.x and now it's missing:

https://github.com/livewire/livewire/blob/2.x/src/Features/SupportRootElementTracking.php#L22